modules/nlu-testing/src/tools/data_loader.ts

```diff
@@ -0,0 +1,135 @@
+import * as sdk from 'botpress/sdk'
+import crypto from 'crypto'
+import _ from 'lodash'
+
+import { BotState, Data, RawData, Test } from '../backend/typings'
+
+async function splitTrainToTrainAndTest(state: BotState) {
```

Comment:
as discussed in person we can't modify the actual train set

Reply:
Changed, we will display only a part of the visualisation if no test datas (e.g no confusion matrix)